tests/e2e/transaction/signTransaction.test.ts

```diff
@@ -1,139 +1,197 @@
       });
       test("it signs an entry function transaction", async () => {
         const rawTxn = await aptos.generateTransaction({
-          sender: senderAccount.accountAddress.toString(),
+          sender: singleSignerED25519SenderAccount.accountAddress.toString(),
           data: {
-            function: `0x${senderAccount.accountAddress.toStringWithoutPrefix()}::transfer::transfer`,
+            function: `0x${contractPublisherAccount.accountAddress.toStringWithoutPrefix()}::transfer::transfer`,
             functionArguments: [new U64(1), recieverAccounts[0].accountAddress],
           },
         });
         const accountAuthenticator = aptos.signTransaction({
-          signer: senderAccount,
+          signer: singleSignerED25519SenderAccount,
           transaction: rawTxn,
         });
         expect(accountAuthenticator instanceof AccountAuthenticator).toBeTruthy();
         const deserializer = new Deserializer(accountAuthenticator.bcsToBytes());
         const authenticator = AccountAuthenticator.deserialize(deserializer);
-        expect(authenticator instanceof AccountAuthenticatorEd25519).toBeTruthy();
+        expect(authenticator instanceof AccountAuthenticatorSingleKey).toBeTruthy();
       });
       test("it signs a multi sig transaction", async () => {
         const rawTxn = await aptos.generateTransaction({
-          sender: senderAccount.accountAddress.toString(),
+          sender: singleSignerED25519SenderAccount.accountAddress.toString(),
           data: {
             multisigAddress: secondarySignerAccount.accountAddress,
-            function: `0x${senderAccount.accountAddress.toStringWithoutPrefix()}::transfer::transfer`,
+            function: `0x${contractPublisherAccount.accountAddress.toStringWithoutPrefix()}::transfer::transfer`,
             functionArguments: [new U64(1), recieverAccounts[0].accountAddress],
           },
         });
         const accountAuthenticator = aptos.signTransaction({
-          signer: senderAccount,
+          signer: singleSignerED25519SenderAccount,
           transaction: rawTxn,
         });
         expect(accountAuthenticator instanceof AccountAuthenticator).toBeTruthy();
         const deserializer = new Deserializer(accountAuthenticator.bcsToBytes());
         const authenticator = AccountAuthenticator.deserialize(deserializer);
-        expect(authenticator instanceof AccountAuthenticatorEd25519).toBeTruthy();
+        expect(authenticator instanceof AccountAuthenticatorSingleKey).toBeTruthy();
       });
     });
-
-    describe.skip("Secp256k1", () => {
+    describe("Single Sender Secp256k1", () => {
+      test("it signs a script transaction", async () => {
+        const rawTxn = await aptos.generateTransaction({
+          sender: singleSignerSecp256k1Account.accountAddress.toString(),
+          data: {
+            bytecode: singleSignerScriptBytecode,
+            functionArguments: [new U64(1), recieverAccounts[0].accountAddress],
```

Comment:
```suggestion
            functionArguments: [new U64(1), receiverAccounts[0].accountAddress],
```

ctrl + h `reciever` and replace with `receiver` :p 😁